README.md

```diff
@@ -170,6 +178,19 @@ Edit file `lib/runInVscode.js` and modify variable `scriptPath` to run your scri
 const scriptPath = `${__dirname}/../examples/got-get.js`;
 ```
 
+### Mock $secure
+Edit file `lib/runInVscode.js` and modify variable `options.secure` to add credentials that are used in your API script, then use `$secure.username` and `$secure.password` to get username and password.
+
+Examples:
+```javascript
+let options = {
+    // mock $secure
+    secure: {
+        username: 'mock username',
+        password: 'mock password'
+    }
+}
+```
```

Comment:
let us not duplicate the secure definitions in different places

Reply:
Since the code is changed, we just need to update the doc as below 

Edit file `lib/runInVscode.js` and modify variable `credFile` in below to specify the credential json path. 

const credFile = `${__dirname}/../examples/creds.json`;